dashboard/app/controllers/api/v1/pd/applications_controller.rb

```diff
@@ -217,6 +214,7 @@ def get_applications_by_role(role, include_associations: true)
       applications_of_type = current_user.workshop_admin? ?
         @applications.where(type: TYPES_BY_ROLE[role].try(&:name)) :
         @applications.where.not(status: 'incomplete').where(type: TYPES_BY_ROLE[role].try(&:name))
```

Comment:
i think you can simplify this line now too, otherwise looks good!

Reply:
Yikes! Good catch